server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/documents.ts

```diff
@@ -162,9 +163,13 @@ export function create(
 				? uuid()
 				: (request.body.id as string) || uuid();
 
+			Lumberjack.info(
+				`Whole summary length= ${JSON.stringify(request.body.summary).length}.`,
```

Comment:
This is too expensive operation to have as logging statement. Same for below.

Reply:
yes, I was planning on removing it, this is just to make sure the new code is being hit.